tests/uninstall.test.zsh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env zsh
+
+# Required for shunit2 to run correctly
+setopt shwordsplit
+SHUNIT_PARENT=$0
+
+# ------------------------------------------------------------------------------
+# SHUNIT2 HOOKS
+# ------------------------------------------------------------------------------
+
+oneTimeSetUp() {
+  export TERM="xterm-256color"
+
+  SPACESHIP_PROMPT_FIRST_PREFIX_SHOW=true
+  SPACESHIP_PROMPT_ADD_NEWLINE=false
+}
+
+setUp() {
+  echo "" > ~/.zshrc
+  ./scripts/install.sh >/dev/null
+}
+
+# ------------------------------------------------------------------------------
+# TEST CASES
+# ------------------------------------------------------------------------------
+
+test_uninstall_preserve_zshrc_content() {
+  echo "TEST=TEST" >> ~/.zshrc
+
+  ./scripts/uninstall.sh -y >/dev/null
+
+  zshrc_content=$(<~/.zshrc)
+
+  assertContains "preserve non spaceship related install config" "$zshrc_content" "TEST=TEST"
+}
+
+test_uninstall_remove_spaceship_content_from_zshrc() {
+  ./scripts/uninstall.sh -y >/dev/null
+
+  zshrc_content=$(<~/.zshrc)
+
+  assertNotContains "remove spaceship load comment" "$zshrc_content" "# Set Spaceship ZSH as a prompt"
+  assertNotContains "remove auto load prompt init" "$zshrc_content" "autoload -U promptinit; promptinit"
+  assertNotContains "remove SPACESHIP_PROMPT_ORDER env" "$zshrc_content" "SPACESHIP_RPROMPT_ORDER"
```

Comment:
We need to add `SPACESHIP_PROMPT_ORDER`  env setting contents before we can assert it

Reply:
Yeah, sorry. I will add this in test.

Reply:
Done in [a42c5c4](https://github.com/denysdovhan/spaceship-prompt/pull/781/commits/a42c5c48efbaa997fd5b9df65df50d7da3eb00b0).